wadas/ui/ai_model_downloader_selector_dialog.py

```diff
@@ -0,0 +1,99 @@
+# This file is part of WADAS project.
+#
+# WADAS is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# WADAS is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with WADAS. If not, see <https://www.gnu.org/licenses/>.
+#
+# Author(s): Stefano Dell'Osa, Alessandro Palla, Cesare Di Mauro, Antonio Farina
+# Date: 2025-01-01
+# Description: Module containing Ai Model downloader selector class and methods.
+
+import os
+from PySide6.QtGui import QIcon
+from PySide6.QtWidgets import (
+    QDialog,
+    QGroupBox,
+    QVBoxLayout,
+    QCheckBox,
+    QScrollArea,
+    QDialogButtonBox,
+)
+
+module_dir_path = os.path.dirname(os.path.abspath(__file__))
+
```

Comment:
Add a blank line (class and function defs should have two blank lines around them).